src/openstack_manager.py

```diff
@@ -113,3 +85,196 @@ def list_projects(cloud_config: dict) -> list[Project]:
         ) from exc
 
     return projects
+
+
+class ImageBuildError(Exception):
+    """Exception representing an error during image build process."""
+
+
+def _build_image_command(
+    runner_info: RunnerApplication, proxies: Optional[ProxyConfig] = None
+) -> list[str]:
+    """Get command for building runner image.
+
+    Returns:
+        Command to execute to build runner image.
+    """
+    http_proxy = proxies.http if (proxies and proxies.http) else ""
+    https_proxy = proxies.https if (proxies and proxies.https) else ""
+    no_proxy = proxies.no_proxy if (proxies and proxies.no_proxy) else ""
+
+    environment = jinja2.Environment(loader=jinja2.FileSystemLoader("templates"), autoescape=True)
+    docker_proxy_service_conf_content = environment.get_template("systemd-docker-proxy.j2").render(
+        proxies=proxies
+    )
```

Comment:
Consider making these separate functions for unit tests rather than having to access index x of the command which might change